server/db/users/create_user.sql

```diff
@@ -0,0 +1,15 @@
+INSERT INTO users
+(authID)
+VALUES 
+(${authid});
+
+INSERT INTO education_table (authid)
```

Comment:
remove inserts on education_table, skills_table, and experience_table. Not needed